### ISSUE TITLE
No way to browse to snapraid.exe location

### ISSUE THREAD

**Issue by Smurf-IV** [BlueBlock/Elucidate#21]
![image](https://user-images.githubusercontent.com/2418812/46255829-259f3a80-c49a-11e8-8a29-998fcbb35601.png)

**Comment by BlueBlock** [comment]
Decision..

1) The snapraid.exe path could be an application setting and not linked to the config or it could be stored in each snapraid.conf file. 

2) Could store the snapraid.exe location within the snapraid.conf file and also as a comment so snapraid.exe ignores it.

Something like:
`###ELUCIDATE-SETTINGS###snapraid-exe c:\snapraid\snapraid.exe`

The setting would be managed by Elucidate. 

Thoughts?

**Comment by Smurf-IV** [comment]
Yes I can see what you are trying to do there.

BTW, _Please do not close the Issues until they have been tested, as no-else but you can then re-open them !_

Also, _Do not forget to Merge changes that I have been doing as well, or just push (Pull request) back into my origin branch_

**Comment by BlueBlock** [comment]
I know you have other projects you are focusing on. Are you planning to put in a lot of time on Elucidate now or focus on the other projects?

**Comment by Smurf-IV** [comment]
yes, I plan to apply more time to this project, it will mostly be at weekends.
